integration-cli/docker_cli_cp_to_container_test.go

```diff
@@ -580,8 +580,10 @@ func (s *DockerSuite) TestCpToErrReadOnlyRootfs(c *check.C) {
 // The `docker cp` command should also ensure that you
 // cannot write to a volume that is mounted as read-only.
 func (s *DockerSuite) TestCpToErrReadOnlyVolume(c *check.C) {
+	// TODO
 	// --read-only + userns has remount issues
-	testRequires(c, DaemonIsLinux, NotUserNamespace)
+	// Windows: non-local daemons for now, skip
```

Comment:
Not quite - this is skipped on Windows as TP4 doesn't support ro volumes, just rw.